ci/on-install.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+const { spawn } = require('child_process');
+const { exit } = require('process');
+
+async function main() {
+  if (fs.existsSync('./build')) {
+    process.stdout.write('-- node-rdkafka bindings already installed, skipping\n');
+    return;
+  }
+
+  const prebuildFileName = `platform-${process.arch}-ABI-${process.versions.modules}`;
```

Comment:
Do we actually need to target the specific ABI here as well? Is this somewhat of a stand-in for `node` version?

I am curious because I wonder if different `node` versions would ever have the same ABI but still cause compatibility issues. I think the answer is no, but double checking here.

Reply:
Yes the ABI is directly linked to how we build and compile the node binary: https://nodejs.org/en/learn/modules/abi-stability

Quoted from the link above:

> APIs provided by the project will result in a stable ABI for all minor and patch versions of Node.js released within one major version